libtransmission/verify.cc

```diff
@@ -274,3 +203,30 @@ tr_verify_worker::~tr_verify_worker()
         std::this_thread::sleep_for(20ms);
     }
 }
+
+int tr_verify_worker::Node::compare(Node const& that) const noexcept
+{
+    // prefer higher-priority torrents
+    if (priority_ != that.priority_)
+    {
+        return priority_ > that.priority_ ? 1 : -1;
+    }
+
+    // prefer smaller torrents, since they will verify faster
+    auto const& metainfo = mediator_->metainfo();
+    auto const& that_metainfo = that.mediator_->metainfo();
+    if (metainfo.total_size() != that_metainfo.total_size())
+    {
+        return metainfo.total_size() < that_metainfo.total_size() ? 1 : -1;
+    }
+
+    // uniqueness check
+    auto const& this_hash = metainfo.info_hash();
+    auto const& that_hash = that_metainfo.info_hash();
+    if (this_hash != that_hash)
+    {
+        return this_hash < that_hash ? 1 : -1;
```

Comment:
*(king of nit picking)* It does not matter, but my OCD says smaller hash values should come first.

Reply:
fixed in 574795b8cf4c4711832efe988b072811ed6795ed